src/main/java/org/eclipse/emfcloud/jackson/databind/type/EcoreTypeFactory.java

```diff
@@ -11,6 +11,7 @@
 
 package org.eclipse.emfcloud.jackson.databind.type;
 
+import static org.eclipse.emf.ecore.EcorePackage.Literals.EBYTE_ARRAY;
```

Comment:
Could you please update the copyright header with the latest year:
`Copyright (c) 2019-2022 Guillaume Hillairet and others.`

Reply:
Done